app/assets/stylesheets/_organizations.scss

```diff
@@ -1,19 +1,18 @@
-div {
-  display: block;
-  float: left
+#wrapper {
+  width: 100%;
+  margin: 0 auto;
+}
+#organizations, #organization-content {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line